Message content intent tag (#2566)
python-discord/bot: bot/resources/tags/mcintent.md

---
embed:
    title: "Message Content Intent"
---

The discord gateway only dispatches events you subscribe to, which you can configure by using "intents"

The message content intent is what determines if an app will receive the actual content of newly created messages

> Disabling or enabling this feature will allow or disallow this data to be received.

Reviewer: This line is superflous
```suggestion

```

Discord has disabled this permission for verified bots (bots verified over 75-100 servers).

> Users can request this permission for their bot.

Author: Fix codeblock.

---
embed:
    title: "Message Content Intent"
---

> The discord gateway only dispatches events you subscribe to, which you can configure by using "intents"

Reviewer 2: ```suggestion
The Discord gateway only dispatches events you subscribe to, which you can configure by using Gateway Intents.
```

> The message content intent is what determines if an app will receive the actual content of newly created messages

Reviewer: I'd just like for this to tie back to the main issue of command handlers not running
```suggestion
The message content intent is what determines if an app will receive the actual content of newly created messages. Without this intent, discord.py wouldn't know which command handler to trigger so none of your commands will work.
```

---
embed:
    title: "Message Content Intent"
---

> Discord regulates data being sent to an application via a specific number.
> 
> This number represents different permissions A.K.A. intents.

Reviewer: I think the explanation of how the gateway intents bitfield is implemented is necessarily important here - we want to explain *how to do something* rather than *how something works*, if that makes sense. So something like
```suggestion
The discord gateway only dispatches events you subscribe to, which you can configure by using "intents"
```

> The message content intent is what determines if an app will receive the content of an ``on_message_create`` event.

Reviewer: ```suggestion
The message content intent is what determines if an app will receive the actual content of newly created messages
```

> Disabling or enabling this feature will allow or disallow this data to be received.

Reviewer: Superfluous after previous line

---
embed:

>     title: "Message Content Intent"

Reviewer 3: ```suggestion
    title: "Discord Bot Message Content Intent"
```
